File: ESFT/Android_Client/src/esft/android/ESFTParameter.java
package esft.android;

public class ESFTParameter {
	// 参数名称
	public String m_ParameterName;

	// 参数内容
	public String m_ParaContent;

	public ESFTParameter() {

	}

	public ESFTParameter(String iParaName, String iParaContent) {
		this.m_ParameterName = iParaName;
		this.m_ParaContent = iParaContent;
	}
}

File: ESFT/ESFT.Common/Message/ESFTParameter.cs

namespace ESFT.Message
{
    /// <summary>
    /// ESFT参数类型
    /// </summary>
    public class ESFTParameter
    {
        string m_parameterName;
        string m_parameterContent;

        /// <summary>
        /// 参数名称
        /// </summary>
        public string ParaName
        {
            get { return this.m_parameterName; }
            set { this.m_parameterName = value; }
        }

        /// <summary>
        /// 参数内容
        /// </summary>
        public string ParaContent
        {
            get { return this.m_parameterContent; }
            set { this.m_parameterContent = value; }
        }

        public ESFTParameter()
        {
        }

        public ESFTParameter(string iParaName, string iParaContent)
        {
            this.m_parameterName = iParaName;
            this.m_parameterContent = iParaContent;
        }
    }
}
